app/models/credence_answer.rb

```diff
@@ -0,0 +1,9 @@
+class CredenceAnswer < ActiveRecord::Base
+  belongs_to :credence_question
+
+  def format
+    # Would be nice to format the text in bold.
+    gen = self.credence_question
```

Comment:
What is gen? Please use a full name.

Also, how is a gen the same thing as a credence question? That is what this line states.